documentation/docfx_project/customize/toc.yml

```diff
@@ -0,0 +1,4 @@
+- name: How to Customize and Use Autoware Packages
+  href: index.html#how-to-customize-and-use-autoware-packages
+- name: Autoware-Mini
+  href: index.html#autoware-mini
```

Comment:
```suggestion
  href: index.html#autoware-mini

```

Reply:
Fixed in https://github.com/AutomotiveAIChallenge/aichallenge2023-sim/pull/25/commits/04fef7b7e599d808379e76a6186c86f164fbdf79